docs/snippets/common/storybook-interactions-play-function.js.mdx

```diff
@@ -0,0 +1,27 @@
+```js
+// MyForm.stories.js
+import { expect } from '@storybook/jest';
+import { userEvent, waitFor, within } from '@storybook/testing-library';
+import { MyForm } from './MyForm'
+
+export default {
+  title: 'MyForm',
+  component: MyForm,
+  argTypes: {
+    onSubmit: { action: true },
+  },
+};
+
+const Template = (args) => <MyForm {...args} />;
+
+const Submitted = Template.bind({});
+Submitted.play = async ({ canvasElement }) => {
```

Comment:
do we want to have this notation in the main docs example? or use `context.canvasElement` instead? if people want to reuse play functions, they must pass the entire context, which they lose when destructuring.

Reply:
```suggestion
Submitted.play = async ({ args, canvasElement }) => {
```